debugger/src/main/java/org/codice/acdebugger/breakpoints/SecurityCheckInformation.java

```diff
@@ -411,26 +413,38 @@ private int getNextContextDomainIndexNotComputedFromStack(List<String> contextDo
     // 'domains' doesn't have nulls and doesn't have duplicates
     final int size = contextDomains.size();
     // skip null at the top since those are not added to 'domains'
-    int nextToFind = (contextDomains.get(0) == null) ? 1 : 0;
+    int nextToFind = findNextToFindNotAlreadyFound(0, contextDomains);
```

Comment:
ℹ️ The updated `findNextToFindNotAlreadyFound()` method accounts for null so that made it much easier to also skip the second or third one if it turns out these were also from the boot domain